File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    // variable declaration & initialisation
    int change = 0, cents = 0;
    float dollars = 0.0;
    // get non negative value of dollars from user
    do
    {
        dollars = get_float("Change Owed : ");
    }
    while (dollars < 0);

    // compute the minimum no. of change required
    cents = round(dollars * 100);
    change = (cents / 25);
    cents = cents % 25;
    change = change + (cents / 10);
    cents = cents % 10;
    change = change + (cents / 5);
    cents = cents % 5;
    change = change + cents;
    printf("%i\n", change);

    return 0;

}

File: pset6/cash/cash.py
# import get_float function from cs50
from cs50 import get_float

# get non negative value of dollars from user
dollars = get_float("Change Owed : ")
while True:
    if dollars < 0:
        dollars = get_float("Change Owed : ")
    else:
        break

# compute the minimum no. of change required
cents = round(dollars * 100, 2)
change = int(cents / 25)
cents = int(cents % 25)
change = change + int(cents / 10)
cents = int(cents % 10)
change = change + int(cents / 5)
cents = int(cents % 5)
change = change + cents
print(change)